### PR TITLE
Make UnicodeDecodeError errors more informative

### PR DESCRIPTION
**Pull request by Jamim**

### CODE REVIEW

**Comment by Jamim** [comment]
CI failure for Python 2.6 looks unrelated to these changes.

**Comment by hugovk** [comment]
The Python 2.6 failure is because wheel has dropped support for it, because 2.6 has been EOL for over 4 years.

**Comment by hugovk** [comment]
I've created PR https://github.com/jdunck/python-unicodecsv/pull/85 to drop support for it and turn the CI green.

File: unicodecsv/py2.py [hunk 1]
```diff
@@ -112,20 +134,28 @@ def __init__(self, f, dialect=None, encoding='utf-8', errors='strict',
...
+            for column, value in enumerate(row, start=1):
+                values.append(
+                    value
+                    if parse_numerics and isinstance(value, float) else
+                    unicode(value, encoding, encoding_errors)
+                )
+        except UnicodeDecodeError as error:
+            error.message = 'Value in cell {}{} can not be decoded. {}'.format(
+                _get_column_name(column), self._current_row_number, str(error)
+            )
```
Comment (ryanhiebert) [inline unicodecsv/py2.py]: This error message seems pretty tied to understanding csv the way it's presented in excel. Perhaps we can say "column X of row Y"?

**Comment by Jamim** [comment]
Closed since this library is [not maintained](https://github.com/jdunck/python-unicodecsv/issues/92) anymore.

**Pull request closed without merging**